typhon/signal.py

```diff
@@ -107,6 +108,11 @@ def signal_widget(signal, read_only=False, tooltip=None):
     widget_instance.setObjectName(name)
     if tooltip is not None:
         widget_instance.setToolTip(tooltip)
+    try:
+        if signal.as_string:
```

Comment:
I'm on the fence here. We could check the `Signal.describe` dictionary to get whether it is a string or not. This would make it less `EpicsSignal` specific. However, this issue seems distinctly specific to `EpicsSignal`. I doubt we will ever have simulated char arrays of strings?

Reply:
TBH I forgot that describe was even thing

Reply:
`describe` returns a `dict` where `'dtype': 'string'`. Kind of awkward relying on a string containing the text `'string'` but I agree it's more compatible across different signals and worth changing

Reply:
Awkward, but we rely on it in a few other places as well, (mainly in the data plugin), and i believe is required for `bluesky` so I at least expect it to be quite stable.

Reply:
Resolving conversation because I changed it